File: C Programming/problems on bitwise operator/problem14.c
/*
14. Write a program which accept one number from user and toggle contents of first and last nibble of the number.
	Return modified number. (Nibble is a group of four bits) 
		
	Author : Milind Nalawade
	
	Date : 03/08/2021
	
	Language : C programming
	
*/
#include<stdio.h>
#include<stdlib.h>

//Logic 
int ChkBit(int no)
{
	int flag=0xf000000f;
	int ino=no|flag;
	return ino;		
}

//Entry point function 
int main(int argc,char * argv[])
{
	int no=atoi(argv[1]);
	int ret=ChkBit(no);
	printf("\n Output : %d \n",ret);
	return 0;
}	

File: Python Programming/problems on bitwise operator/problem14.py
'''
13. Write a program which accept one number and position from user and on that bit. Return modified number. 
	
	  Input :  10 4
	  Output :  14 
	
	Author : Milind Nalawade
	
	Date : 03/08/2021
	
	Language : Python 3.9.0

'''
import sys

#====
#Logic
#====
def ChkBit(no):
	flag=0xf000000f
	ino=no|flag
	print (ino)
	return ino;
		
#=============
#Entry point function
#=============
def main():
	no=int(sys.argv[1])
	ret=int(ChkBit(no))
	print("\n Output : {} \n".format(ret));
	
#=====
#starter
#=====	
if __name__=="__main__":
	main()